### PR TITLE
Added communitybridge button and consolidated GSoC 2020 and 2019 descriptions.

### LINKED ISSUE #328: [Internship Programs] Replace old GSoC with CommunityBridge
**Current State of the Meshery.io site**
Meshery participates in CommunityBridge - https://people.communitybridge.org/project/0d6fd362-04a1-4086-a6e7-ec753ed4a60b. CommunityBridge is missing from this list:

Screen Shot 2020-09-24 at 2 22 58 PM

**Desired State of the Meshery.io site**
1. add communitybridge button to the list
1. Consolidate GSoC 2020 and 2019. A single GSoC page should hold both 2020 and 2019 descriptions.

---
**Contributor Resources**
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md.

### PR DESCRIPTION
**Pull request by Sudhish2101**
**Description**

This PR fixes #328

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Neilblaze — APPROVED** [review]
LGTM :+1:

File: collections/_programs/gsoc_2020.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 <div class="cover" style="z-index: 20">
   <h3 class="black-text title">
-    Google Summer of Code 2020 and Layer5 Community
+    Google Summer of Code (2020 & 2019) and Layer5 Community
```
Comment (JubayerJoy) [inline collections/_programs/gsoc_2020.html:14]:
```suggestion
    Google Summer of Code and Layer5 Community
```
As Layer5 will take part in more GSoC, best to not add years.

**Review by JubayerJoy — CHANGES_REQUESTED** [review]
Really nice work @Sudhish2101 🚀 
Some minor changes are required.

**Comment by JubayerJoy** [comment]
@leecalcote, @Nikhil-Ladha 
Should the consolidated GSoC page need a `side-navbar` for each year?

File: collections/_programs/gsoc_2020.html (hunk 1 above)
Comment (Nikhil-Ladha) [reply collections/_programs/gsoc_2020.html:14]: Agreed

File: _data/programs.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@
   new_window: false
 
 
-- name: Google Summer of Code 2020
+- name: Google Summer of Code
   imagePath: /images/programs/gsoc.svg
   link: /programs/gsoc/2020
```
[the remark this answers is not in the document]
Comment (Nikhil-Ladha) [reply _data/programs.yml:15]: No, this can be left as `gsoc/2020` only so as to add future pages wrt to years.

**Comment by Nikhil-Ladha** [comment]
> @leecalcote, @Nikhil-Ladha
> Does GSoC page needs a `side-navbar` for each year?
> 
> I think it'll be great.

Yeah, either a sidenav bar or the links can be added to the bottom of the page.

**Comment by Neilblaze** [comment]
@Nikhil-Ladha are we participating in MLH fellowship? If no, then MLH2020 card should be removed from [here](https://deploy-preview-360--mesheryio.netlify.app/programs/)

**Comment by Neilblaze** [comment]
But again 3x2 will lose the symmetry, so yeah it's a bit confusing over here.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha are we participating in MLH fellowship? If no, then I think it'll be better if we remove MLH2020 card from [here](https://deploy-preview-360--mesheryio.netlify.app/programs/).

@leecalcote ?

File: collections/_programs/gsoc_2020.html (hunk 1 above)
Comment (Sudhish2101) [reply collections/_programs/gsoc_2020.html]: Done

File: _data/programs.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
   new_window: false
```
[the remark this answers is not in the document]
Comment (Sudhish2101) [reply _data/programs.yml]: Done

File: _data/programs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sudhish2101) [reply _data/programs.yml:9]: As suggested by @Nikhil-Ladha, I have not changed this.

**Comment by Sudhish2101** [comment]
Does it look good?

**Comment by JubayerJoy** [comment]
LGTM 👍 
Really nice work @Sudhish2101

**Review by Nikhil-Ladha — APPROVED** [review]
Nice work @Sudhish2101 👍🏼

**Pull request merged**